### PR TITLE
Feature/add unittesting

### PR DESCRIPTION
**Pull request by yash-puligundla**

### CODE REVIEW

**Comment by yash-puligundla** [comment]
- added unittests
- CI pipeline
(We will have to configure travis, coverall, and change the links in readme after merging to master)

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Travis (.org) branch](https://app.travis-ci.com/yash-puligundla/refget-compliance-suite.svg?branch=feature%2Fadd_unittesting)](https://app.travis-ci.com/yash-puligundla/refget-compliance-suite) [![Coverage Status](https://coveralls.io/repos/github/yash-puligundla/refget-compliance-suite/badge.svg?branch=feature/add_unittesting)](https://coveralls.io/github/yash-puligundla/refget-compliance-suite?branch=feature/add_unittesting) [![Read the Docs](https://img.shields.io/badge/docs-passing-brightgreen.svg)](http://samtools.github.io/hts-specs/refget.html) [![PyPI](https://img.shields.io/badge/pypi-v0.1.0-blue)](https://pypi.org/project/refget/) [![Python 3.6|3.7](https://img.shields.io/badge/python-3.6%20|%203.7-blue.svg?)](https://www.python.org)
```
Comment (jb-adams) [inline README.md:1]: As we discussed, please place each of these badges on their own line for readability

Comment (jb-adams) [inline README.md:1]: Also, please point the build status and coverage status images to the `master` branch of the `ga4gh` fork. Thanks!

**Review by jb-adams — CHANGES_REQUESTED** [review]
Looks great, just about ready to merge! As a final point, there are 2 files introduced to version control that can be removed:
1. unittests/.DS_Store
2. unittests/data/.DS_Store

I think the `.DS_Store` entry in `.gitignore` was added after these got tracked. Can you remove these 2 using `git rm`?

**Comment by yash-puligundla** [comment]
Good catch. 

- Deleted the .DS_Store files
- Updated the coverage and travis links to point to ga4gh/refget-compliance-suite master

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Travis (.org) branch](https://app.travis-ci.com/ga4gh/refget-compliance-suite.svg?branch=master
+)](https://app.travis-ci.com/github/ga4gh/refget-compliance-suite)
```
Comment (jb-adams) [inline README.md:3]: looks like an unnecessary newline was introduced here?

**Review by jb-adams — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (yash-puligundla) [reply README.md]: removed the new line

**Review by jb-adams — APPROVED** [review]

**Pull request merged**